astropy/io/fits/hdu/image.py

```diff
@@ -979,6 +979,12 @@ def shape(self):
         # `ImageHDU.section` in place of `.data`.
         return self.hdu.shape
 
+    @property
+    def dtype(self):
+        # Implementing `.dtype` enables `astropy.nddata.Cutout2D` to accept
+        # `ImageHDU.section` in place of `.data`.
+        return np.dtype(BITPIX2DTYPE[self.hdu._bitpix])
```

Comment:
This doesn't take scaling into account, so I think the correct way would be:
```suggestion
        return self.hdu._dtype_for_bitpix()
```